runtime/doc/lsp.txt

```diff
@@ -699,6 +699,24 @@ buf_detach_client({bufnr}, {client_id})          *vim.lsp.buf_detach_client()*
       • {bufnr}      (integer) Buffer handle, or 0 for current
       • {client_id}  (integer) Client id
 
+buf_inlay_hint_disable({bufnr})             *vim.lsp.buf_inlay_hint_disable()*
```

Comment:
Why doesn't this live in `vim.lsp.buf.*` like almost all other buffer-relative LSP features? Strange that it has a `buf_` prefix instead.

Reply:
I'll move it; I was suggested `buf_`

Reply:
still wondering where the `buf_` suggestion came from..

Reply:
It came from clason